community/php7/APKBUILD

```diff
@@ -157,203 +322,285 @@ build() {
 		--enable-phpdbg \
 		--enable-phpdbg-webhelper \
 		--disable-cgi \
-		--disable-cli \
-		|| return 1
+		--disable-cli
+
 	# apache2 module
 	_build --disable-phpdbg \
 		--disable-cgi \
 		--disable-cli \
-		--with-apxs2 \
-		|| return 1
-	mv libs/libphp7.so sapi/apache2handler/mod_php7.so
-	# cgi,cli,fpm,embed,litespeed
+		--with-apxs2
+	mv libs/libphp$_suffix.so sapi/apache2handler/mod_php$_suffix.so
+
+	# cgi, cli, fpm, embed, litespeed
 	_build --disable-phpdbg \
 		--enable-fpm \
 		--enable-embed \
-		--with-litespeed \
-		|| return 1
+		--with-litespeed
+}
+
+check() {
+	cd "$builddir"
+
+	# PHP is so stupid that it's not able to resolve dependencies
+	# between extensions and load them in correct order, so we must
+	# help it...
+	# * opcache is Zend extension, it's handled specially in Makefile
+	# * imap can't be loaded together with recode, so we must omit it
```

Comment:
Yeah, even tests are okay, many of them are skipped because of “could not connect to mailbox {localhost/norsh}” (WTF?!), but this is a different issue. But it means that we need to explicitly define order not based on dependencies, omfg…